library/src/main/scala/scala/scalajs/js/special/package.scala

```diff
@@ -231,4 +231,6 @@ package object special {
   def debugger(): Unit =
     throw new java.lang.Error("stub")
 
+  private[scalajs] def linkTimeProperty[T](name: String): T =
+    throw new java.lang.Error("stub")
```

Comment:
I think we should put this into `scalajs.LinkingInfo` instead. It's supposed not to be JS-dependent.